services/vision/server.go

```diff
@@ -87,13 +87,31 @@ func detsToProto(detections []objectdetection.Detection) []*pb.Detection {
 		yMin := int64(box.Min.Y)
 		xMax := int64(box.Max.X)
 		yMax := int64(box.Max.Y)
-		d := &pb.Detection{
-			XMin:       &xMin,
-			YMin:       &yMin,
-			XMax:       &xMax,
-			YMax:       &yMax,
-			Confidence: det.Score(),
-			ClassName:  det.Label(),
+
+		normbox := det.NormalizedBoundingBox()
+		var d *pb.Detection
+		if len(normbox) == 4 {
+			d = &pb.Detection{
+				XMin:           &xMin,
+				YMin:           &yMin,
+				XMax:           &xMax,
+				YMax:           &yMax,
+				Confidence:     det.Score(),
+				ClassName:      det.Label(),
+				XMinNormalized: &normbox[0],
+				YMinNormalized: &normbox[1],
+				XMaxNormalized: &normbox[2],
+				YMaxNormalized: &normbox[3],
+			}
+		} else {
+			d = &pb.Detection{
+				XMin:       &xMin,
+				YMin:       &yMin,
+				XMax:       &xMax,
+				YMax:       &yMax,
+				Confidence: det.Score(),
+				ClassName:  det.Label(),
```

Comment:
Mild preference to put most of this back the way it originally was, so it's not so redundant.
```go
d := &pb.Detection{
    XMin: ...
    YMin: ...
    XMax: ...
    YMax: ...
    Confidence: ...
    ClassName: ...
}
normbox := ...
if len(normbox) == 4 {
    d.XMinNormalized = &normbox[0]
    d.YMinNormalized = &normbox[0]
    ...
}
```